### PR TITLE
T.herman/cylinder tex coords

### PR DESCRIPTION
**Pull request by tomasz-herman**

### CODE REVIEW

**Comment by tomasz-herman** [comment]
In the end i used original texturing, same as was in opengl part, except its mirrored along u-axis in cylinder part
Screenshot_20201231_202100

File: RayTracer/Source/Models/Cylinder.cs [hunk 1]
```diff
@@ -72,6 +73,7 @@ private void CalculateBottomAndTop()
         {
             _bottom = -_height * 0.5f * Vector3.UnitY * RotationMatrix + Position;
             _top = _height * 0.5f * Vector3.UnitY * RotationMatrix + Position;
+            _center = (_bottom + _top) / 2;
             _normal = (_top - _bottom) / _height;
```
Comment (karolk98) [inline RayTracer/Source/Models/Cylinder.cs:77]:
```suggestion
            _center = Position;
            _normal = Vector3.UnitY * RotationMatrix;
```

File: RayTracer/Source/Models/Cylinder.cs [hunk 2]
```diff
@@ -72,6 +73,7 @@ private void CalculateBottomAndTop()
         {
```
Comment (karolk98) [inline RayTracer/Source/Models/Cylinder.cs:73]: Maybe rename to Recalculate() or sth

File: RayTracer/Source/Renderer/Renderer.cs [hunk 3]
```diff
@@ -21,7 +21,7 @@ public void Render(Scene scene, Camera camera)
             foreach (var model in scene.Models)
             {
                 if (!model.Loaded) continue;
-                _shader.SetMatrix4("model", model.GetModelMatrix());
+                // _shader.SetMatrix4("model", model.GetModelMatrix());
```
Comment (p-flis) [inline RayTracer/Source/Renderer/Renderer.cs:24]: not needed anymore, commit https://github.com/tomasz-herman/RayTracing/commit/67c3c43912dab62689603785fe463b1f50c43883 fixes the problem

**Pull request merged**